certbot/cli.py

```diff
@@ -686,7 +675,7 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
               "which ones are required if it finds one missing")
     helpful.add(
         None, "--dialog", dest="dialog_mode", action="store_true",
-        help="Run using interactive dialog menus")
+        help="Deprecated - this has no effect")
```

Comment:
I wonder if literally anyone ever used the `--dialog` flag, I have no problem deprecating that one properly.

As for `--text`, we have a whole bunch of tests that use it, I think suppressing it is the right move.